lib/git.js

```diff
@@ -45,8 +45,8 @@ async function isRefInHistory(ref) {
 /**
  * Unshallow the git repository (retriving every commits and tags).
  */
-async function unshallow() {
-  await execa('git', ['fetch', '--unshallow', '--tags'], {reject: false});
+async function unshallow(origin) {
```

Comment:
Can you rename the `origin` parameter to `repositoryUrl` and add the JSDoc?

Reply:
done